frontend/apps/marketing/src/selectors/contentful/getImage.ts

```diff
@@ -4,8 +4,23 @@ export function getRelativeImageUrl(asset: ExperienceAsset | undefined) {
   return asset?.fields?.file?.url;
 }
 
-export function getAbsoluteImageUrl(asset: ExperienceAsset | undefined) {
-  const relativeImageUrl = getRelativeImageUrl(asset);
+export function getAbsoluteImageUrl(
+  asset: ExperienceAsset | string | undefined,
+) {
+  const imgUrl =
+    typeof asset === 'string' ? asset : getRelativeImageUrl(asset) || undefined;
 
-  return relativeImageUrl ? `https:${relativeImageUrl}` : undefined;
+  if (!imgUrl) return undefined;
+
+  try {
+    const absoluteImgUrl = new URL(
+      imgUrl.startsWith('//') ? `https:${imgUrl}` : imgUrl,
+    );
+    // Force AVIF format conversion for the image
+    absoluteImgUrl.searchParams.set('fm', 'avif');
+    return absoluteImgUrl.toString();
+  } catch (e) {
+    console.error(e);
```

Comment:
Non-blocking: @dmcavoy - do we want to inform content editors in anyway of invalid images?

Reply:
Ya I think that would be good. What make something an invalid image?